mypy/errors.py

```diff
@@ -618,8 +618,12 @@ def report_internal_error(err: Exception,
 
     # Print "INTERNAL ERROR" message.
     print('{}error: INTERNAL ERROR --'.format(prefix),
-          'please report a bug at https://github.com/python/mypy/issues',
-          'version: {}'.format(mypy_version),
+          'Please try using master\n'
+          'https://mypy.rtfd.io/en/latest/common_issues.html#using-pre-release-mypy',
```

Comment:
This link doesn't seem to point to the correct section (it goes to the top of the page).

Reply:
Ah, I forgot to commit a change to shorten the title of that section so the link isn't egregious.